tickets/src/config.js

```diff
@@ -107,7 +109,8 @@ export default function configure(
     // See https://www.reddit.com/r/ethereum/comments/3c8v2i/what_is_the_expected_block_time/
     blockTime = 8000
 
-    unlockAppUrl = 'https://unlock-protocol.com/'
+    unlockAppUrl = 'https://app.unlock-protocol.com'
+    unlockStaticUrl = 'https://unlock-protocol.com'
```

Comment:
Waiiiit. Aren't we setting these thru env vars???

Reply:
I think we should remain consistent with the rest of the app (see `next.config.js` because it's already such a mess). Can we remove all hard coded values (but dev) here?